index.html

```diff
@@ -86,7 +87,7 @@ <h2>Canopy</h2>
           <li><button class="custom-buttonds" type="button">css</button></li>
           <li><button class="custom-buttonds" type="button">JavaScript</button></li>
         </ul>
-        <button id="buttonds1" class="big-buttonds" type="button">See Project</button>
+        <button id="buttonds1" class="big-buttonds project-button" type="button"  onclick="populatePopup(0);">See Project</button>
       </div>
     </div>
```

Comment:
- [ ] <b>REQUIRED:</b> The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript. Check an exampe of students projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95 and https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96.

Reply:
As you can see I used an Array where I stored all the card information (objects) then I use a javascript function to load every time different card, I don't really understand what can I do more?